.circleci/docker/ubuntu-cuda/Dockerfile

```diff
@@ -62,12 +62,22 @@ RUN if [ -n "${VISION}" ]; then bash ./install_vision.sh; fi
 RUN rm install_vision.sh
 ENV INSTALLED_VISION ${VISION}
 
+# (optional) Install non-default Ninja version
+ARG NINJA_VERSION
```

Comment:
we are planning to extend `install_llvm` to use ninja if available, in future?

Reply:
Artifact from an earlier version of the patch.  I'll eliminate it.